packages/public-api/src/web.ts

```diff
@@ -0,0 +1,52 @@
+export interface MaskNetworkAPIs {
+    web_echo<T>(arg: T): Promise<T>
+    /**
+     * @returns A fully quantified URL in forms of "holoflows-extension://...." or "moz-extension://...." (based on the platform)
+     */
+    getDashboardURL(): Promise<string>
+
+    /**
+     * @deprecated Use settings_getLaunchPageSettings()
+     */
+    getSettings(key: 'launchPageSettings'): Promise<LaunchPage>
+
+    /**
+     * @returns A stringified JSON string.
+     * @example [[{ network: "twitter.com", connected: false } ]]
+     */
+    getConnectedPersonas(): Promise<string>
+    app_isPluginEnabled(pluginID: string): Promise<boolean>
+    app_setPluginStatus(pluginID: string, enabled: boolean): Promise<void>
+    setting_getNetworkTraderProvider(network: NetworkType): Promise<TradeProvider | undefined>
+    setting_setNetworkTraderProvider(network: NetworkType, provider: TradeProvider): Promise<void>
+    settings_getTrendingDataSource(): Promise<DataProvider>
+    settings_setTrendingDataSource(provider: DataProvider): Promise<void>
+    settings_getLaunchPageSettings(): Promise<LaunchPage>
+}
+export enum LaunchPage {
```

Comment:
i want's enable this eslint rule (`@typescript-eslint/prefer-enum-initializers`).

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-enum-initializers.md

Reply:
It doesn't always necessary. We only need to explicitly mark enum values in cases it is a public API.